src/lib/receptions/SequencescapeTubes.js

```diff
@@ -0,0 +1,99 @@
+import {
+  fetchLabwareFromSequencescape,
+  findIncluded,
+  getIncludedData,
+  buildRequestAndSample,
+} from './sequencescapeUtils.js'
+// This module replaces by services.Sequencescape which can be removed
+// when the pipeline-specific receptions are retired. While this change results
+// in temporary code duplication, it allows for complete decoupling of old and
+// new paths, greatly simplifying the removal.
+
+/**
+ * Request parameters for retrieval of labware from Sequencescape v2 API
+ */
+const labwareRequestConfig = {
+  include: 'receptacles.aliquots.sample.sample_metadata,receptacles.aliquots.study',
+  fields: {
+    tubes: 'labware_barcode,receptacles',
+    wells: 'position,aliquots',
+    receptacles: 'aliquots',
+    samples: 'sample_metadata,name,uuid',
+    sample_metadata: 'sample_common_name',
+    studies: 'uuid',
+    aliquots: 'study,library_type,sample',
+  },
+}
+
+/**
+ *
+ * @param {Object} labware Labware object from Sequencescape
+ * @param {Array<Object>} included Included objects from Sequencescape
+ * @param {Object} requestOptions Additional request parameters, will over-ride any
+ * @returns {Object} tubes_attributes object ready for import into traction
+ */
+const transformTube = ({ labware, included, requestOptions }) => {
+  // find the receptacle in the included data
+  const receptacle = findIncluded({
+    included,
+    data: labware.relationships.receptacles.data[0],
+    type: 'receptacles',
+  })
+
+  // get the aliquot, study, sample and sample_metadata from the included data
+  const { aliquot, study, sample, sample_metadata } = getIncludedData({
+    labware: receptacle,
+    included,
+  })
+
+  return {
+    barcode: labware.attributes.labware_barcode.human_barcode,
+    // build the request and sample objects
+    ...buildRequestAndSample({ aliquot, study, sample, sample_metadata, requestOptions }),
+  }
+}
+const labwareTypes = {
```

Comment:
labwareTypes seem to be the same in both receptions. Maybe we can move it to the utils?

Reply:
Actually they are different in both -
For Sequencescape it is, 
`const labwareTypes = {
  plates: {
    type: 'plates',
    attributes: 'plates_attributes',
    transformFunction: transformPlate,
  },
  tubes: {
    type: 'tubes',
    attributes: 'tubes_attributes',
    transformFunction: transformTube,
  },
}
`
For SequenceTubes , it is
`const labwareTypes = {
  plates: {
    type: 'plates',
    attributes: 'plates_attributes',
    transformFunction: transformPlate,
  },
  tubes: {
    type: 'tubes',
    attributes: 'tubes_attributes',
    transformFunction: transformTube,
  },
}
`

Probably , we can have the default in utils as 
`const labwareTypes = {
  tubes: {
    type: 'tubes',
    attributes: 'tubes_attributes',
    transformFunction: transformTube,
  },
}
`
and in Sequencescape , we can modify this generic tye to append the 'plates' as well. But , I am not sure whether it improves much as there are only two of them. Please comment.

Reply:
Could the default have both plates and tubes and then each reception uses the appropriate one?

Reply:
That sounds good. I will  make the changes  👍